test/libsolidity/smtCheckerTests/abi/abi_encode_packed_simple.sol

```diff
@@ -23,10 +23,6 @@ contract C {
 }
 // ----
 // Warning 6328: (354-384): CHC: Assertion violation happens here.
-// Warning 1218: (451-481): CHC: Error trying to invoke SMT solver.
-// Warning 6328: (451-481): CHC: Assertion violation might happen here.
+// Warning 6328: (451-481): CHC: Assertion violation happens here.
 // Warning 6328: (560-590): CHC: Assertion violation happens here.
-// Warning 1218: (609-639): CHC: Error trying to invoke SMT solver.
-// Warning 6328: (609-639): CHC: Assertion violation might happen here.
-// Warning 4661: (451-481): BMC: Assertion violation happens here.
-// Warning 4661: (609-639): BMC: Assertion violation happens here.
+// Warning 6328: (609-639): CHC: Assertion violation happens here.
```

Comment:
Note to self: Change in expectation is possibly unrelated to this PR in that the test case does not contain transaction variables as such.

Reply:
It is true that the test case does not touch transaction variables, but the constraints are nevertheless added in the encoding. So the encoding with this PR is different than the previous encoding. Basically, Spacer sees (syntactically) different problem. And this is known to cause different behaviour in Spacer; sometimes in our favour, sometimes in the wrong direction...